apps/cowswap-frontend/src/modules/hooksStore/dapps/AirdropHookApp/index.tsx

```diff
@@ -45,55 +37,83 @@ export function AirdropHookApp({ context }: HookDappProps) {
 
   return (
     <Wrapper>
-      <Header>
-        <img src={IMAGE_URL} alt={NAME} width="120" />
-        <p>{DESCRIPTION}</p>
-      </Header>
       <ContentWrapper>
         <Row>
+          <LabelContainer>
+            <label>Select Airdrop</label>
+          </LabelContainer>
+          {/* <label style={{ width: 'fit-content' }}>Select Airdrop</label> */}
           <DropDownMenu airdropOptions={AIRDROP_OPTIONS} setSelectedAirdrop={setSelectedAirdrop} />
```

Comment:
Now I reallize that we're not filtering by chain as well (since we always tested on sepolla). Could you filter out airdrops of other chains? 

Also, I think that each airdrop option information should be for just one chain.

Reply:
Also would be good to add a message if there isn't any airdrop on that chain